e2etests/orchestration/common.go

```diff
@@ -186,3 +191,23 @@ func UploadAndExtract(srv orchclient.HostOrchestratorService, remoteDir, src str
 	}
 	return nil
 }
+
```

Comment:
A comment describing how the file is verified would be nice.

Reply:
Done.